pom.xml

```diff
@@ -50,8 +50,8 @@
         <apache.httpclient.version>4.5.2</apache.httpclient.version>
         <bouncycastle.bcpkix.version>1.55</bouncycastle.bcpkix.version>
         <confluent.maven.repo>http://packages.confluent.io/maven/</confluent.maven.repo>
-        <jersey.version>2.25</jersey.version>
-        <jetty.version>9.2.19.v20160908</jetty.version>
+        <jersey.version>2.26-b06</jersey.version>
+        <jetty.version>9.4.6.v20170531</jetty.version>
```

Comment:
I understand... how about leaving the project as is for JDK7, and a `jdk8` classifier for JDK8?  (And the same for downstream projects.)

This way, nothing downstream has to change if they wanna keep using JDK7.